Spec/RestClientPresence.swift

```diff
@@ -75,6 +75,48 @@ class RestClientPresence: QuickSpec {
                     }
                 }
 
+                // RSP3a3
+                pending("connectionId should filter members by the provided connectionId") {
+                    let options = AblyTests.commonAppSetup()
+                    let client = ARTRest(options: options)
+                    let channel = client.channels.get("test")
+
+                    var disposable = [ARTRealtime]()
+                    defer {
+                        for clientItem in disposable {
+                            clientItem.close()
+                        }
+                    }
+
+                    waitUntil(timeout: testTimeout) { done in
+                        disposable += AblyTests.addMembersSequentiallyToChannel("test", members: 6, options: options) {
+                            done()
+                        }
+                    }
+
+                    waitUntil(timeout: testTimeout) { done in
+                        disposable += AblyTests.addMembersSequentiallyToChannel("test", members: 3, startFrom: 7, options: options) {
+                            done()
+                        }
+                    }
+
+                    let query = ARTRealtimePresenceQuery()
+                    query.connectionId = disposable.last!.connection.id!
```

Comment:
Mind adding a comment or explaining why the connection ID of the last disposable only returns the last 3?